bin/MadGraph5_aMCatNLO/cards/production/13TeV/HeavyNeutrino_trilepton_V-0p01_e_NLO/HeavyNeutrino_trilepton_mNgtmWltmZ_V-0p01_e_NLO/HeavyNeutrino_trilepton_V-0p01_e_NLO_proc_card.dat

```diff
@@ -0,0 +1,25 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set loop_color_flows False
+set gauge unitary
+set complex_mass_scheme False
+set max_npoint_for_channel 0
+import model sm
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
+define l+ = e+
+define ldecay+ = e+ mu+ ta+
+define l- = e-
+define ldecay- = e- mu- ta-
+define l = l+ l-
+define ldecay = ldecay+ ldecay-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+define v = vl vl~
+define w = w- w+
```

Comment:
wouldn't it be cleaner to remove all this info and define the particles you really need only after you imported your model?